### DISCUSSION TITLE
Huge pp drop after playing just a beatmap

### DISCUSSION THREAD

**Discussion by rvnexe** [ppy/osu#23863] [Q&A] [answered]
Hi. my pp was about 167. I played a beatmap (I got A with about 12 pp) and then my pp droped to about 40. 
Is this because ScoreV2 or bug?

**Comment by Sarisan** [comment, marked as answer]
I had the same issue https://github.com/ppy/osu/discussions/23846

**Reply by rvnexe** [reply]
Cheers.
